packages/manager/modules/pci/src/projects/project/storages/databases/add/add.html

```diff
@@ -255,7 +255,7 @@ <h2 id="network" data-translate="pci_database_add_step6_title"></h2>
                             data-ng-if="$ctrl.availableSubnets.length > 1"
                             data-match="name"
                             data-searchable
-                            data-on-change="$ctrl.getOrderData()"
+                            data-on-change="$ctrl.getOrderDataFromModel($ctrl.model)"
```

Comment:
Same as the comment on `onAdditionalDiskChange`, it is not needed anymore.

Reply:
Fixed :)